superdesk/utils.py

```diff
@@ -74,6 +75,13 @@ def get_random_string(length=12):
     return str(uuid4())
 
 
+def get_random_token(length=40,
```

Comment:
Could we use just
```
>>> uuid.uuid4().hex
'7193d651965d499496c9e8f439e8a9ce'
```
it has 32 length, `uuid4` is usually the best way to get a random string (more secure as I know)